dashboard/src/deployment/DeploymentList.js

```diff
@@ -1,48 +1,67 @@
 import React, { Component } from 'react';
 import { apiGet } from '../api/api.js';
-import DeploymentRow from './DeploymentRow.js';
-import { Table } from 'semantic-ui-react';
-//import logo from './logo.svg';
-//import './App.css';
+import { Icon, Table } from 'semantic-ui-react';
+import Loading from '../util/Loading.js';
+
+const HeaderView = () => (
+  <Table.Header>
+    <Table.Row>
+      <Table.HeaderCell>State</Table.HeaderCell>
+      <Table.HeaderCell>Name</Table.HeaderCell>
+      <Table.HeaderCell>Mode</Table.HeaderCell>
+      <Table.HeaderCell>Pods</Table.HeaderCell>
+    </Table.Row>
+  </Table.Header>
+);
+
+const RowView = ({info}) => (
```

Comment:
I'd pass in the item's properties individually to make the props more explicit here.